command/sub8_alarm/alarm_handlers/kill.py

```diff
@@ -10,27 +12,33 @@ class Kill(HandlerBase):
     def __init__(self):
         self._killed = False
         self._last_mission_killed = False
-        self.bagging_server = rospy.ServiceProxy('/online_bagger/dump', BaggerCommands)
+        self.bag_client = SimpleActionClient('/online_bagger/bag', BagAction)
```

Comment:
Me and @mattlangford decided to just hard code this topic everywhere, because it is used in several places across multiple projects